apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -75,8 +79,9 @@ var (
 		},
 		"v1.9": {
 			KubeAPI: map[string]string{
-				"endpoint-reconciler-type": "lease",
-				"admission-control":        "ServiceAccount,NamespaceLifecycle,LimitRanger,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds",
+				"endpoint-reconciler-type":                "lease",
+				"admission-control":                       "ServiceAccount,NamespaceLifecycle,LimitRanger,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds",
+				"experimental-encryption-provider-config": "/etc/kubernetes/encryption.yaml",
```

Comment:
As user can opt out of passing encryption config, we have to make sure that passing `experimental-encryption-provider-config` would work fine when SecretsEncryptionConfig is nil on the api service.